src/controllers/playback/video/index.js

```diff
@@ -1035,26 +1040,46 @@ import { appRouter } from '../../../components/appRouter';
             const key = keyboardnavigation.getKeyName(e);
             const isKeyModified = e.ctrlKey || e.altKey || e.metaKey;
 
+            const btnPlayPause = osdBottomElement.querySelector('.btnPause');
+
             if (e.keyCode === 32) {
                 if (e.target.tagName !== 'BUTTON' || !layoutManager.tv) {
                     playbackManager.playPause(currentPlayer);
+                    showOsd(btnPlayPause);
                     e.preventDefault();
                     e.stopPropagation();
                     // Trick Firefox with a null element to skip next click
                     clickedElement = null;
+                } else {
+                    showOsd();
                 }
-                showOsd();
                 return;
             }
 
+            if (layoutManager.tv && !currentVisibleMenu) {
+                // Change the behavior of some keys when the OSD is hidden
+                switch (key) {
+                    case 'ArrowLeft':
+                    case 'ArrowRight':
+                        showOsd(nowPlayingPositionSlider);
+                        nowPlayingPositionSlider.dispatchEvent(new KeyboardEvent(e.type, e));
```

Comment:
As an alternative to the event constructor polyfill, we can use a wrapper to create an event.